github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.30.yaml

```diff
@@ -392,11 +392,19 @@ presubmits:
             topologyKey: kubernetes.io/hostname
       containers:
       - command:
-        - /usr/local/bin/runner.sh
-        - /bin/sh
-        - -c
-        - |
-          apt update && apt install gettext-base -y && wget https://dl.google.com/go/go1.13.8.linux-amd64.tar.gz && tar -xzf go1.13.8.linux-amd64.tar.gz -C /usr/local/ && export PATH=/usr/local/go/bin:$PATH && export TARGET=kind-k8s-sriov-1.17.0 && automation/test.sh
+        - "/usr/local/bin/runner.sh"
+        - "/bin/bash"
```

Comment:
If we add -xe here we don't need the && in the script

Reply:
Will it affect the `trap`?